scripts/load_osf.py

```diff
@@ -0,0 +1,25 @@
+import requests
+from pathlib import Path
+
+data1 = ("data1.csv", "https://osf.io/yzqf8/download/")
```

Comment:
please use informative dataset names

Reply:
I have updated the names, see https://github.com/laurens88/FORAS_composing_datasets/pull/2/commits/9f743c51042249c1f68d525ce073f3b5895ea634